pc2_to_laserscan_cpp:
  ros__parameters:
    input_cloud: "/velodyne_pointcloud"
    output_laserscan: "/velodyne_pointcloud/scan"
    angle_min: -3.14
    angle_max: 3.14
    angle_increment: 0.0058
    z_min: -1.0
    z_max: 2.0
    range_min: 0.05
    range_max: 50.0
    use_inf: false
    scan_time: 1.0

